style.css

```diff
@@ -534,120 +534,135 @@
 
 /* == cantact part start == */
 
-.card-4 {
+.contact-form-container {
+  margin: 0 auto;
   display: flex;
   flex-direction: column;
-  flex-wrap: wrap;
-  align-content: center;
-  width: 327px;
+  width: 375px;
+  text-align: center;
 }
 
-.contact-text-card {
+.contact-form-container p {
   width: 327px;
-  margin-top: 110px;
+  height: 216px;
+  margin-top: 120px;
+  margin-bottom: 58px;
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
+  font-size: 31px;
   line-height: 44px;
-  color: #172b4d;
   text-align: center;
+  color: #172b4d;
 }
 
-.contact-field-card {
+form input {
+  height: 48px;
   width: 327px;
-  height: 200px;
-  background: #fff;
+  margin: 24px 0;
```

Comment:
- [ ] Good job implementing the contact form, as mentioned in the project requirements **you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).** let's also make the design follow the figma template as well currently you have extra margin between input fields please see :point_down: please fix this
![image](https://user-images.githubusercontent.com/97521557/214650302-bc717938-04b7-4ff2-be04-b94fad98f344.png)

- [ ] As mentioned in the project requirements **'In order to lay out the elements on the page you should use Flexbox.'**  I suggest you use flexbox for your form's layout as it is much more responsive and gives you more control,